core/tools/nginx/nginx.conf

```diff
@@ -195,9 +195,24 @@ http {
             try_files $uri $uri/ /index.html;
         }
 
+        location ~ ^/upload(/.*)$ {
+            client_max_body_size 100M;
+            alias /usr/blueos$1;
+
+            # Change the access permissions of the uploaded file
+            dav_access group:rw all:r;
+            create_full_put_path on;
+
+            # Configure the allowed HTTP methods
+            dav_methods PUT DELETE MKCOL COPY MOVE;
+        }
+
         location /userdata {
             root /usr/blueos;
             autoindex on;
+            # why not json?
```

Comment:
what ?

Reply:
This makes nginx output json instead of html for autoindex

Reply:
The comment should be more explicit, your explanation does not reflect the comment

Reply:
done